tests/e2e/suite.go

```diff
@@ -153,33 +157,196 @@ func (s *TestSuite) TestQueryBaseFee() {
 	})
 }
 
-func (s *TestSuite) TestSendTxUpdating() {
-	ctx := context.Background()
+func (s *TestSuite) TestSendTxDecrease() {
+	// cast chain to cosmos-chain
+	cosmosChain, ok := s.chain.(*cosmos.CosmosChain)
+	s.Require().True(ok)
+	// get nodes
+	nodes := cosmosChain.Nodes()
+	s.Require().True(len(nodes) > 0)
+
+	params := s.QueryParams()
+
+	baseFee := s.QueryBaseFee()
+	gas := int64(200000)
+	minBaseFee := baseFee.MulDec(math.LegacyNewDec(gas))[0]
+	minBaseFeeCoins := sdk.NewCoins(sdk.NewCoin(minBaseFee.Denom, minBaseFee.Amount.TruncateInt()))
+	sendAmt := int64(100000)
+
+	s.Run("expect fee market state to decrease", func() {
+		s.T().Log("performing sends...")
+		for {
+			// send with the exact expected fee
+
+			wg := sync.WaitGroup{}
+			wg.Add(3)
+
+			go func() {
+				defer wg.Done()
+				txResp, err := s.SendCoinsMultiBroadcast(
+					context.Background(),
+					s.user1,
+					s.user2,
+					sdk.NewCoins(sdk.NewCoin(cosmosChain.Config().Denom, math.NewInt(sendAmt))),
+					minBaseFeeCoins,
+					gas,
+					1,
+				)
+				s.Require().NoError(err, txResp)
+				s.Require().Equal(uint32(0), txResp.CheckTx.Code, txResp.CheckTx)
+				s.Require().Equal(uint32(0), txResp.TxResult.Code, txResp.TxResult)
+			}()
+
+			go func() {
+				defer wg.Done()
+				txResp, err := s.SendCoinsMultiBroadcast(
+					context.Background(),
+					s.user3,
+					s.user2,
+					sdk.NewCoins(sdk.NewCoin(cosmosChain.Config().Denom, math.NewInt(sendAmt))),
+					minBaseFeeCoins,
+					gas,
+					1,
+				)
+				s.Require().NoError(err, txResp)
+				s.Require().Equal(uint32(0), txResp.CheckTx.Code, txResp.CheckTx)
+				s.Require().Equal(uint32(0), txResp.TxResult.Code, txResp.TxResult)
+			}()
+
+			go func() {
+				defer wg.Done()
+				txResp, err := s.SendCoinsMultiBroadcast(
+					context.Background(),
+					s.user2,
+					s.user3,
+					sdk.NewCoins(sdk.NewCoin(cosmosChain.Config().Denom, math.NewInt(sendAmt))),
+					minBaseFeeCoins,
+					gas,
+					1,
+				)
+				s.Require().NoError(err, txResp)
+				s.Require().Equal(uint32(0), txResp.CheckTx.Code, txResp.CheckTx)
+				s.Require().Equal(uint32(0), txResp.TxResult.Code, txResp.TxResult)
+			}()
+
+			wg.Wait()
+			fee := s.QueryBaseFee()
+			s.T().Log("base fee", fee.String())
+
+			if fee.AmountOf(feemarkettypes.DefaultFeeDenom).Equal(params.MinBaseFee) {
```

Comment:
What does this check do?

Reply:
we are checking that the fee has decreased and been "clamped" to the min base fee